docs/launcher.rst

```diff
@@ -0,0 +1,101 @@
+.. _Launcher:
+
+Launcher
+=========
+
+On this page we describe the different launchers for experiments. It is possible to use the local
+machine or batch schedulers like `Slurm <https://slurm.schedmd.com/overview.html>`_ or
+`Oracle Grid Engine <https://docs.oracle.com/cd/E19680-01/html/821-1541/docinfo.html#scrolltoc>`_
+(formerly SGE) to run experiments.
+
+In the following sections, we consider the
+`sorting expample <https://github.com/hu-macsy/simexpal/tree/master/examples/sorting>`_ from the
+:ref:`QuickStart` guide.
+
+..
+    TODO: Add section on ForkLauncher, Slurm, SGE
+
+Queue Launcher
+--------------
+
```

Comment:
Add some motivation for the queue launcher. It is mainly useful in scenarios where no sophisticated batch scheduler is available, especially when launching experiments on a remote machine over `ssh.`. In contrast to the fork launcher, it does not require an active connection.